.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,26 @@
+# This action creates a release the first Monday each month at 5:00 UTC.
+name: "Create and push release tag"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "* 5 * * 1"
```

Comment:
I think the previous version was correct.  This one is "every minute, at 5 AM, every day, every month, if it's a Monday".

That first bit would be especially weird. :)

Given that this wont be changing often and our workflow skips releases when there aren't changes, let's switch it to every Monday and do it on the same weekly frequency as osbuild/images: `0 5 * * 1`.

Reply:
> I think the previous version was correct. This one is "every minute, at 5 AM, every day, every month, if it's a Monday".

Ah flip that was supposed to be a `zero` at the front not a star. Silly typo. But yeah we can simplify and just go with the same frequency as images